doc/content/lorawan/radio-propagation/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Radio Propagation"
+section: Additional Information
+description: ""
+weight:
+---
+
+Radio propagation describes how the radio signals travel from the transmitter to the receiver inside the earth's atmosphere or free space.
+
+## Line of Sight
+
+Line of Sight (LoS) is a type of propagation that can transmit and receive data only where the transmitter and receiver are in view of each other without any obstacles between them. Even in the LoS, the signal strength gets reduced when it passes through the air for a long distance.
```

Comment:
The first sentence is not really correct. LoS propagation is a type of propagation where signal waves travel along direct transmitter-receiver path without any obstacles. This definition doesn't have anything to do with transceiver's ability to transmit and receive data. Some transceivers can transmit/receive only if LoS exists, while others can do it if LoS doesn't exist, i.e. if there's obstacles between them. 

This part "a type of propagation that can transmit and receive data ..." is also not correct - the propagation cannot transmit or receive anything.